D2Go/gradle.properties

```diff
@@ -0,0 +1,28 @@
+ABI_FILTERS=armeabi-v7a,arm64-v8a,x86,x86_64
+
+VERSION_NAME=0.0.1-SNAPSHOT
+GROUP=org.pytorch
+MAVEN_GROUP=org.pytorch
+POM_URL=https://github.com/pytorch/vision/
+POM_SCM_URL=https://github.com/pytorch/vision.git
+POM_SCM_CONNECTION=scm:git:https://github.com/pytorch/vision
+POM_SCM_DEV_CONNECTION=scm:git:git@github.com:pytorch/vision.git
+POM_LICENSE_NAME=BSD 3-Clause
+POM_LICENSE_URL=https://github.com/pytorch/vision/blob/master/LICENSE
+POM_ISSUES_URL=https://github.com/pytorch/vision/issues
+POM_LICENSE_DIST=repo
+POM_DEVELOPER_ID=pytorch
+POM_DEVELOPER_NAME=pytorch
+syncWithMavenCentral=true
+
+GRADLE_BINTRAY_PLUGIN_VERSION=1.8.0
+GRADLE_VERSIONS_PLUGIN_VERSION=0.15.0
+ANDROID_MAVEN_GRADLE_PLUGIN_VERSION=2.1
```

Comment:
Can be removed.
those properties are needed only for publishing releases/nightlies.